### PR TITLE
libvips: remove redundant libpng dependency

### PR DESCRIPTION
**Pull request by kleisauke**
In favor of libspng, which is preferred by libvips.

Depends on: https://github.com/libvips/libvips/pull/3919.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
@lovell @jcupitt Do you approve?

**Comment by lovell** [comment]
@jonathanmetzman This PR is dependent on https://github.com/libvips/libvips/pull/3919 - I can approve after it has been merged.

**Comment by jonathanmetzman** [comment]
OK do you approve now?

**Review by lovell — APPROVED** [review]

**Pull request merged**
